docs/conf.py

```diff
@@ -8,13 +8,15 @@
 project = 'HDL'
 copyright = '2024, Analog Devices, Inc.'
 author = 'Analog Devices, Inc.'
+version = '0.5' # documentation version, will be printed on the cover page
```

Comment:
Don't, we should inject [here](https://github.com/analogdevicesinc/doctools/blob/main/adi_doctools/__init__.py#L59-L65) a value if not set instead.
An empty string `""` also causes issues?

The whole idea is to set by CI instead, to not have to commit version changes + we use semester release format.

Reply:
will test this out

Reply:
No, an empty string won't affect the building process. The only thing is that on the cover page the version is missing when building locally. I suppose when this is deployed and the version is injected, when building the doc the version will appear. I will leave it like this:
`version = ''`